<launch>
<node pkg="test_actions" type="server.py" name="number_sequence_server" output="screen"/>
</launch>
